### DISCUSSION TITLE
I have not been able to install the plugin

### DISCUSSION THREAD

**Discussion by DonBrokoLee** [simonoliver/UnityFigmaBridge#33] [General]
Hello friend, I tried to install the plug in, and it gives me this error
![Screenshot 2023-09-09 at 11 01 45 a m](https://github.com/simonoliver/UnityFigmaBridge/assets/102316910/22fc4e2d-eeac-45cb-bf14-40f2b611dda0)

You have any idea what i have to do?

**Comment by simonoliver** [comment]
It looks like you don't currently have git installed on your Mac (this is needed to install packages from github). 

If you are familiar with the command line, you can type the following in terminal:

xcode-select --install

More info here:

https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
